codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/ActionServiceSourceGenerator.scala

```diff
@@ -383,7 +383,7 @@ object ActionServiceSourceGenerator {
       |  }
       |
       |  @Override
-      |  public final ActionHandler newHandler(ActionCreationContext context) {
+      |  public final ${service.handlerName} newHandler(ActionCreationContext context) {
```

Comment:
we could have this as `ActionHandler<$classNameAction>` but that wasn't how other components were generated so I aligned with the others